### PR TITLE
Eli Kalish, Assignment 1

### PR DESCRIPTION
**Pull request by elik723**
I hope I did this right.

### CODE REVIEW

File: assignment1.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+1. What is the significance of return 0; at the end of the main program? Is it  really needed?
+The main function needs to return a number because ANSI standards dictate that
+main() must be preceded by int. The function returns 0 becasue that indicates   nothing went wrong during the course of main. None of this is really neccesary  for the code to function, it's just a standard put in place.
```
Comment (oldclesleycode) [inline assignment1.txt:3]: Great answer, Eli! 💯

File: assignment1.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
+1. What is the significance of return 0; at the end of the main program? Is it  really needed?
+The main function needs to return a number because ANSI standards dictate that
+main() must be preceded by int. The function returns 0 becasue that indicates   nothing went wrong during the course of main. None of this is really neccesary  for the code to function, it's just a standard put in place.  
+2. What, if anything is wrong with the following code?
+int main() {
+	float x = 41.5;
+	printf("The value of x is %d", x);
+}
+In the printf statement, %d is being used to try and represent something that isnot an integer variable. %f should be used to represent x because it is a float 
+value. There is also not a return statement, which is neccesary since there is
```
Comment (oldclesleycode) [inline assignment1.txt:10]: Yup! You're missing just one more error - see if you can find it!

File: assignment1.txt [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+int preceding main().
+3. What, if anything, is wrong with this implementation of the equation
+g = -G m/r^2 (the gravitational field equation)?
+	g = -G * m/ r * r
+The value would not be computed correctly because due to order of operations,
+-G would be multiplied by m, then divided by r, then multiplied by r, which     
+isn't the correct order of operations. A quick fix is to put the r * r in
```
Comment (oldclesleycode) [inline assignment1.txt:17]: Yup! You're missing a few more things - see if you can find them!

File: assignment1.txt [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+int preceding main().
+3. What, if anything, is wrong with this implementation of the equation
+g = -G m/r^2 (the gravitational field equation)?
+	g = -G * m/ r * r
+The value would not be computed correctly because due to order of operations,
+-G would be multiplied by m, then divided by r, then multiplied by r, which     
+isn't the correct order of operations. A quick fix is to put the r * r in       
+parentheses.
+4. What is the difference between ++var and var++?
+Depending one whether the operand is a prefix or postfix the order of operationsis changed. If the operand is postfix, the order is from right to left. If the  operand is prefix, the order is from left to right.
```
Comment (oldclesleycode) [inline assignment1.txt:20]: Great answer! 💯

File: assignment1.txt (hunk 2 above)
Comment (elik723) [inline assignment1.txt:10]: Is it the #include <stdio.h> statement?

Comment (oldclesleycode) [inline assignment1.txt:10]: Yup! Since this code prints output, you'd need #include <stdio.h> at the very top! Nice job 👍

File: EK.c [hunk 5]
```diff
@@ -0,0 +1,10 @@
+/* Eli Kalish, Prints my Initials */
```
Comment (oldclesleycode) [inline EK.c:1]: Perfect code!! 💯  Great job! 👍

**Pull request closed without merging**